en_ewt-ud-train.conllu

```diff
@@ -151906,7 +151906,7 @@
 8	Fire	Fire	PROPN	NNP	Number=Sing	4	obl	4:obl:about	_
 9	but	but	CCONJ	CC	_	11	cc	11:cc	_
 10-11	that's	_	_	_	_	_	_	_	_
-10	that	that	PRON	DT	Number=Sing|PronType=Dem	4	nsubj	4:nsubj	_
+10	that	that	PRON	DT	Number=Sing|PronType=Dem	13	nsubj	13:nsubj	_
```

Comment:
"but", "'s", "not" should also be headed by "what"

Reply:
yes, I'll fix that, thanks!